tests/functions/test_lde.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from zospy.functions.lde import find_surface_by_comment
+
+
+class TestLDEFunctions:
```

Comment:
Imo a test class is not needed here (classes are purely used for grouping tests, and grouping at file level seems sufficient here).

Reply:
I have reverted it to a single function